include/loops/loops.hpp

```diff
@@ -90,6 +92,13 @@ enum {
     OP_BREAK,
     OP_CONTINUE,
 
+//Intel-only operations:
+    OP_CQO, //TODO(ch): I don't think, that there will be need to direct use of this command, so it must be in some service sector, actually.
+    OP_XCHG,
+//Aarch64-only operations:
+    OP_CINC,
```

Comment:
this is ARM-specific instruction; please, use OP_ARM_CINC

Reply:
Done.